pkg/codegen/internal/test/testdata/.gitignore

```diff
@@ -2,3 +2,7 @@ package-lock.json
 go.sum
 go.mod
 */go/tests
+*/python/tests
```

Comment:
These kinds of files are copied from python-extras; ignore them in `python/` folders that do get checked in.